src/components/DataTypes/date-time.tsx

```diff
@@ -14,6 +18,11 @@ const getDefaultDate = (): string => {
 	return `${today}T00:00`;
 };
 
+// Normalize a timestamp to a RFC3339 timestamp, which is required for JSON
+// schema.
+// http://momentjs.com/docs/#/displaying/format/#default-format
```

Comment:
This link doesn't work for me, I think you want https://momentjs.com/docs/#default-format